### PR TITLE
Add KnockKnock.app v1.5.0

### PR DESCRIPTION
**Pull request by koenrh**

### CODE REVIEW

File: Casks/knockknock.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'knockknock' do
+  version '1.5.0'
+  sha256 'c0b4c3d8d7fb9981998f6b305aff792b3120c27c978c77cb7484f9a7799f793a'
+
+  url "https://bitbucket.org/objective-see/deploy/downloads/KnockKnock_#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/knockknock.rb]: Needs a `# bitbucket.org is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

File: Casks/knockknock.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  name 'KnockKnock'
+  homepage 'https://objective-see.com/products/knockknock.html'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/knockknock.rb]: `:gratis`.

**Comment by koenrh** [comment]
@vitorgalvao: Thank you. Fixed!

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as d7a31f704708b1de204a309c37e8497cbd2dc97d to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**